src/vma/sock/sockinfo_tcp.cpp

```diff
@@ -721,7 +721,7 @@ ssize_t sockinfo_tcp::tx(const tx_call_t call_type, const struct iovec* p_iov, c
 
 	if (unlikely(is_dummy) && !check_dummy_send_conditions(flags, p_iov, sz_iov)) {
 		unlock_tcp_con();
-		errno = EIO;
+		errno = EOPNOTSUPP;
```

Comment:
@liranoz12 explained that purpose of this check mostly relate logic flow. Probably `EAGAIN` errno is suitable here.